webapp/static/js/argus/drawtree.js

```diff
@@ -1861,7 +1870,7 @@ ArgusNode.prototype.isActualLeafNode = function() {
     return this.num_tips === 0;
 };
 ArgusNode.prototype.isVisibleLeafNode = function() {
-    return this.hasChildren === false || this.num_tips === 0;
+    return this.hasChildren === false || this.num_tips_in_view === 0;
```

Comment:
Minor correction to my commit message. This is used to detect **visible** tips, not "true" tips from the synthetic tree.